pandas/tseries/tests/test_resample.py

```diff
@@ -619,6 +621,52 @@ def test_resample_interpolate(self):
             df.resample('1T').asfreq().interpolate(),
             df.resample('1T').interpolate())
 
+    def test_resample_empty_series(self):
+        # GH12771 & GH12868
+
+        s = self.create_series()[:0]
+
+        for freq in ['M', 'D', 'H']:
+            methods = resample_methods.copy()
+            # need to test for ohlc from GH13083
+            methods.remove('ohlc')
+            for method in methods:
+                expected_index = s.index
+
+                result = getattr(s.resample(freq), method)()
+                expected = s
+                assert_index_equal(result.index, expected_index)
```

Comment:
can u explicitly compare `.freq` also, as `assert_index_equal` doesn't (i'll submit another PR once this is fixed).